### PR TITLE
[AMS-314] Divide conflict status codes

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -11,10 +11,7 @@ stages:
 before_script:
   - export APPLICATION=`make -s print-APPLICATION`
   - source ${CI_PROJECT_DIR}/ci-starter-kit/utils.sh
-  - export BRANCH_NAME=`echo ${CI_BUILD_REF_NAME} | tr '[:upper:]' '[:lower:]' | sed 's/_/-/g' | sed -e 's/[^a-z0-9-]//g'`
-  - export GIT_SHA=`git rev-parse --short=8 HEAD`
-  - export CI_DEV_TAG="dev-${BRANCH_NAME}-${GIT_SHA}"
-  - export CI_TAG=`[[ -z ${CI_BUILD_TAG} ]] && echo "branch-${BRANCH_NAME}-${GIT_SHA}" || echo "${CI_BUILD_TAG}"`
+  - export CI_TAG=`[[ -z ${CI_BUILD_TAG} ]] && echo "branch-${CI_BUILD_REF_SLUG}-${CI_BUILD_REF:0:8}" || echo "${CI_BUILD_TAG}"`
```
Comment (xakep139) [inline .gitlab-ci.yml]: https://docs.gitlab.com/ce/ci/variables/README.html#9-0-renaming

**Comment by denisivan0v** [comment]
@xakep139 take another look at https://github.com/2gis/nuclear-vstore/pull/41/commits/164f9998e95cd056f4fa5876934d9eb662eeacce

File: src/VStore.Host/ActionResults/GoneResult.cs [hunk 2]
```diff
@@ -16,6 +18,7 @@ public GoneResult(DateTime expiresAt) : base(410)
         public override void ExecuteResult(ActionContext context)
         {
             base.ExecuteResult(context);
+            context.HttpContext.Response.ContentType = ContentType.PlainText;
```
Comment (xakep139) [inline src/VStore.Host/ActionResults/GoneResult.cs]: Response doesn't contain any text here, do we need to set ContentType here?

Comment (denisivan0v) [inline src/VStore.Host/ActionResults/GoneResult.cs]: agreed, setting content type removed

**Pull request merged**